### ISSUE TITLE
Incorrect type warning on "symbol in object" expression

### ISSUE THREAD

**Issue by rconnamacher** [google/closure-compiler#3060]
Closure Compiler's type warnings show an incorrect type error when using a `symbol` on the left side of an `in` expression:

```
src/js/SourceFile:100: WARNING - left side of 'in'
found   : symbol
required: string
        if ( SYMBOL in record ) {
              ^^^^^^^^^
```

This should be updated to allow either a string or symbol on the left side of `in`.

(Using `--compilation_level=ADVANCED_OPTIMIZATIONS`, `--language_out=ECMASCRIPT5_STRICT`, and Closure Compiler version `20180805`

**Comment by concavelenz** [comment]
I added support for symbol as a primitive type to the type system and type checker, but I clearly missed "in".   It should be a trivial fix.

**Comment by concavelenz** [comment]
Thanks for the report. I have a fix out for review.
